day_7/10_little_monkeys.rb

```diff
@@ -0,0 +1,26 @@
+class Rhyme
```

Comment:
This is a solid attempt Daniel! I'm glad that you broke this challene up into classes and methods, because that's what you will do with all of your projects in Mod-1. Can you think of a data structure that you could use that was a collection of integers that ll pointed to their string word equivalent? Part of the challenge here is getting the program to print words instead of integers, so 'one' instead of 1. Keep working on this and the other Day-7 challenges today. I recommend time boxing each challenge to about 1-2 hours so that you can practice with a time constraint, which you will see weekly in Mod-1.